core/templates/components/graph-services/graph-layout.service.ts

```diff
@@ -50,18 +50,18 @@ interface NodeData {
   depth: number;
   offset: number;
   reachable: boolean;
-  y0: number;
-  x0: number;
-  yLabel: number;
-  xLabel: number;
-  height: number;
-  width: number;
-  id: string;
-  label: string;
+  y0: number | null;
+  x0: number | null;
+  yLabel: number | null;
+  xLabel: number | null;
+  height: number | null;
+  width: number | null;
+  id: string | null;
+  label: string | null;
```

Comment:
Why can these be `null`?

Reply:
Yeah, that is the explanation of usage, but not of the purpose, why are these set to `null` at that place? Is it possible to not use `null`? If not then it should be documented and explained what it means.